melodies_monet/driver.py

```diff
@@ -11,6 +11,7 @@
 import datetime
 
 # from util import write_ncf
+import melodies_monet
```

Comment:
```suggestion
```
Suggest removing this and importing the function or module you need within the method that needs it. For example

```python
from .util.tools import vert_interp
```

Reply:
Made suggested changes